src/yaml.cr

```diff
@@ -147,4 +164,21 @@ module YAML
   def self.dump(object, io : IO)
     object.to_yaml(io)
   end
+
+  # Checks to see if the value is reserved
+  def self.reserved_value?(value, checks = 0)
```

Comment:
The `checks` attribute is never used and should be removed.

Reply:
The return value of this method should be coerced to `Bool` instead of a union including `Nil`, `Time`, `Float64` and `Int64`.